apps/storybook/.storybook/main.ts

```diff
@@ -8,6 +8,7 @@ const config: StorybookConfig = {
     "@storybook/experimental-addon-test",
     "@storybook/addon-a11y",
     "@storybook/addon-designs",
+    "storybook-addon-test-codegen",
```

Comment:
let's move this addon up after addon-test
```suggestion
    "storybook-addon-test-codegen",
    "@storybook/addon-a11y",
    "@storybook/addon-designs",
```